tox.ini

```diff
@@ -13,17 +13,43 @@ deps =
     pytest-subtests
 extras =
     {env:TEST_EXTRAS_TOX:}
-setenv = COVERAGE_FILE={toxworkdir}/.coverage.{envname}
+setenv =
+    COVERAGE_FILE={toxworkdir}/.coverage/.coverage.{envname}
```

Comment:
I'm guessing this is one of those improvements not tied to the C extension ... but just in case, be sure the changes don't mess up the testing of the python version.

Reply:
The coverage changes are related to the C extension because they are part of adding coverage for the C extension to this repo, which involves a reorganization of where the coverage goes. I could do that in two steps (reorg the Python then the C), but the reasoning for the Python reorg would be lost.